cognite/powerops/client/shop/shop_run_api.py

```diff
@@ -175,3 +197,50 @@ def retrieve(
             )
         else:
             raise TypeError(f"Invalid type {type(external_id)} for external_id.")
+
+    def retrieve_latest(
+        self,
+        watercourse: str | list[str] | None = None,
+        source: str | list[str] | None = None,
+        start_after: str | arrow.Arrow | datetime.datetime | None = None,
+        start_before: str | arrow.Arrow | datetime.datetime | None = None,
+        end_after: str | arrow.Arrow | datetime.datetime | None = None,
+        end_before: str | arrow.Arrow | datetime.datetime | None = None,
+        sort_property: Literal[
```

Comment:
```suggestion
        latest_by: Literal[
```
The sorting is the underlying implementation should not be exposed to the enduser.